lib/modulr/resources/payments/payment.rb

```diff
@@ -64,7 +66,7 @@ def initialize(raw_response, attributes = {}, opts = { network_scheme: true })
           detail_type = @attr_details&.dig(:type) || @attr_details&.dig(:destinationType)
 
           case detail_type
-          when "PI_SECT", "PI_SEPA_INST", "PI_FAST", "PI_REV"
+          when "PI_SECT", "PI_SEPA_INST", "PI_FAST", "PI_REV", "PO_REV"
```

Comment:
It is something that has sense, since a reversal is like an incoming payment to the origin account so that is why we need to parse its attributes as we already do for incoming payments.